tests/scripts/command/prepare-model

```diff
@@ -14,9 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-COMMAND_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-INSTALL_DIR="$(dirname $(dirname $COMMAND_DIR))"
-
```

Comment:
By removing these variables and defining them in `onert_test`, `prepare-model` seems not standalone script any longer. Do you want to make `onert-test` as main driver and others as subcommands?

Reply:
Right. Scripts under `command` don't have execute permission now.